packages/@aws-cdk/pipelines/lib/codepipeline/codebuild-step.ts

```diff
@@ -197,6 +197,15 @@ export class CodeBuildStep extends ShellStep {
    */
   readonly timeout?: Duration;
 
+  /**
+   * Information about logs for the build project.
+   *
+   * A project can create logs in Amazon CloudWatch Logs, an S3 bucket, or both.
+   *
+   * @default - no log configuration is set
+   */
+  readonly logging?: codebuild.LoggingOptions;
```

Comment:
My comment above may mean refactoring `LoggingOptions` instead of building a new interface for this specifically. In fact, that's probably the best path forward. We could improve this experience in general.